api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -281,6 +284,13 @@ message GrpcStatusFilter {
   bool exclude = 2;
 }
 
+// Filters based on dynamic metadata 'access_log_policy' key. The filter will not log if the key is set to `false`.
+// If the key is set to `true` or unset, the request will be logged.
+message LogKeyFilter {
+  option (udpa.annotations.versioning).previous_message_type =
+      "envoy.config.filter.accesslog.v2.LogKeyFilter";
```

Comment:
Why wouldn't we make this a general purpose dynamic metadata filter, and just say "use envoy.acess_log_policy" if you want that capability. Seems more generally useful.

Reply:
Done.